cantera/build_py.bat

```diff
@@ -20,6 +20,7 @@ IF ERRORLEVEL 1 EXIT 1
 IF ERRORLEVEL 1 EXIT 1
 
 ROBOCOPY "%SRC_DIR%\samples\python" "%PREFIX%\share\cantera\samples\python" /S /E
+ROBOCOPY "%SRC_DIR%\platform\posix\man" "%PREFIX%\share\man\man1" /S /E
```

Comment:
Does Windows understand man files?

Reply:
Probably not. Although Cantera installs them :wink:

Deleting.